gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.java.home = /Library/Java/JavaVirtualMachines/jdk1.8.0_202.jdk/Contents/Home
```

Comment:
what if the other engineer has a different path?

Reply:
In my intellij idea settings , I missed to set Gradle JVM , to point to JAVA_HOME. So , I was facing an issue , each time had to run the 'export JAVA_HOME=/path'  command in project root folder . I was using mac os with intellij idea IDE. This is fixed in my local environment and hence I remove gradle.properties which is not required.